creator-node/src/services/stateMachineManager/stateReconciliation/recoverOrphanedData.jobProcessor.ts

```diff
@@ -325,3 +327,30 @@ const _getPrimaryForWallet = async (
   }
   return replicaSet[0]
 }
+
+async function recoverOrphanedData(
+  params: DecoratedJobParams<RecoverOrphanedDataJobParams>
+) {
+  const { parentSpanContext } = params
+  const jobProcessor = instrumentTracing({
+    name: 'recoverOrphanedData.jobProcessor',
+    fn: _recoverOrphanedData,
+    options: {
+      links: parentSpanContext
+        ? [
+            {
+              context: parentSpanContext
+            }
+          ]
+        : [],
+      attributes: {
+        [tracing.CODE_FILEPATH]: __filename
+      }
+    }
+  })
+
+  return await jobProcessor(params)
+}
+
+export default recoverOrphanedData
```

Comment:
do we need both?

Reply:
Ideally we'd have one but I've been keeping the `module.exports` in files to be compatible with other js files

Reply:
and this doesn't cause any issues?